Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStore.cpp

```diff
@@ -755,6 +755,90 @@ void JitArm64::stmw(UGeckoInstruction inst)
     gpr.Unlock(ARM64Reg::W0);
 }
 
+// Sets up the idle loop for common cache loop patterns, in the following form:
```

Comment:
This isn't an idle loop. An idle loop is a loop that does nothing and that doesn't exit until disturbed by some outside force.

This also applies to Jit64.